app/uk/gov/hmrc/helptosavefrontend/forms/BankDetails.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavefrontend.forms
+
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.libs.json.{Format, Json}
+
+object BankDetails {
+  def giveBankDetailsForm(): Form[BankDetails] = Form(
+    mapping("sortCode" -> nonEmptyText(6, 6),
+      "accountNumber" -> nonEmptyText(maxLength = 10),
+      "rollNumber" -> optional(text),
+      "accountName" -> nonEmptyText(maxLength = 50)
```

Comment:
added some dummy validation for now,  do we have an agreed field lengths for each field above ?